invenio_checks/components.py

```diff
@@ -121,9 +120,9 @@ def _run_checks(self, identity, data=None, record=None, errors=None, **kwargs):
                     latest_check.result = res.to_dict()
 
                 self.uow.register(ParentRecordCommitOp(record))
-            except Exception:
+            except Exception as e:
                 current_app.logger.exception(
-                    "Error running check on record",
+                    "Error running check on record: {}".format(e),
```

Comment:
This is not good for Sentry, since we'll start getting ungrouped errors. In theory `current_app.logger.exception(...)` should show the stacktrace in the terminal logs during local dev.